cx-sig-charter-draft.md

```diff
@@ -0,0 +1,110 @@
+  - Low-risk changes impact a small number (<4) of SIGs, working groups, or repositories, do not break existing contributor workflows, and are easy to roll back. When implementing low-risk changes we:
+    - Socialize on <community mailing list>@googlegroups.com and our weekly update calls
+    - We will go to each lead, their mailing lists, slack channel, and/or their update meetings and ask for feedback and a [lazy consensus] process. We will follow up with a post to [<need to figure out what the main mailing list is>@]googlegroups.com mailing list
+  - High-risk changes impact a large number (>4) of SIGs, working group, or repositories, break existing contributor workflows and are not easy to roll back. When implementing high-risk changes we:
+    - Socialize on <we will need community mailing list> and our weekly update calls
+    - Seek [lazy consensus] with a time box of at least 72 business hours with a GitHub issue link (or proposal if not applicable) to the following mailing lists:
+        - [<mailinglist>@]googlegroups.com
+        - <sig lead mailing list>@googlegroups.com
+        - [<need to figure out what the main mailing list is>@]googlegroups.com with the GitHub issue link including the subject [NOTICE]: $announcement
+        - We will also announce it at the weekly Spinnaker Community Meeting on <we will need to decide on a day>
+- Depending on how wide of an ecosystem change this is, we may also slack, blog, tweet, and use other channels to get the word out.
+- Our standard time box is 72 business hours; however, there may be situations where we need to act quickly but the time period will always be clear and upfront.
+- Encourage automation to improve productivity for contributors where it makes sense and consults with SIG Testing if automation is covering GitHub workflows.
+
+If we need funding for any reason, we approach [CDF].
+CDF in many of the noted cases above contributes funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. In these cases, such as Slack, there is at least two Contributor Experience [communication] subproject OWNERs listed as admins.
+
+### Out of scope
+
+- Code for the testing and CI infrastructure - that’s SIG Testing
+- [spinnaker/community]  ownership of folders for RFCs and Design Proposals. Members are to follow those folder’s owner’s files and SIG leadership for the specific issue/PR in question.
+- User community management. We hold office hours because contributors are a large portion of the volunteers that run that program.
```

Comment:
```suggestion
- User community management.
```

@prophen and @ExitoLab, this is your call. No one else is paying attention to our user community so it may make sense to adjust scope here. Curious to hear your thoughts.